resources/views/gallery/_image.blade.php

```diff
@@ -22,7 +22,7 @@
             </div>
         </div>
     @else
-        @if ($image->isFont())
+        @if ($image->isFont() || $image->isFile())
```

Comment:
use !$image->hasThumbnail() instead as discussed above, so it's a single if instead of two.